js/src/pages/price-benchmark/price-benchmark-tour.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { TourKit } from '@woocommerce/components';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useTour from '~/hooks/useTour';
+import PriceBenchmarkGraphic from '~/images/price-benchmark/tour.svg';
+
+const TOUR_ID = 'price-benchmark-tour';
+
+/**
+ * Renders the tour for notifying the new feature of price benchmark & suggestions.
+ */
+export default function PriceBenchmarkTour() {
+	const { tourChecked, setTourChecked } = useTour( TOUR_ID );
+
+	if ( tourChecked ) {
+		return null;
+	}
+
+	const config = {
+		steps: [
+			{
+				referenceElements: {
+					desktop: '.gla-price-benchmark__card a[id="suggestions"]',
+				},
+				meta: {
+					heading: (
+						<>
+							<img
+								className="gla-price-benchmark-tour__graphic"
+								src={ PriceBenchmarkGraphic }
+								alt={ __(
+									'Graphic showing the price benchmark and suggestions feature',
+									'google-listings-and-ads'
+								) }
+							/>
+							{ createInterpolateElement(
```

Comment:
@ankitrox There's no need to use `createInterpolateElement` here.
```suggestion
							<span className="gla-tour-highlight">Price Benchmark & Suggestions</span>
```